starter-code/public/js/script.js

```diff
@@ -0,0 +1,19 @@
+
+
+$( "#image-switch" ).click( () => {
```

Comment:
This is cool. Maybe a more elegant way would have been to hold the paths of images in an array and just fetch the next one on click? :)